creator-node/src/middlewares.js

```diff
@@ -40,7 +40,9 @@ async function authMiddleware (req, res, next) {
       attributes: ['blockchainId'],
       where: { cnodeUserUUID }
     })
-    userId = parseInt(resp.blockchainId)
+    if (resp && resp.blockchainId) {
+      userId = parseInt(resp.blockchainId)
+    }
```

Comment:
what if still !userId at this point?

Reply:
it'll allow it to pass, but if ensurePrimary is called then it will reject (which is correct). i didn't reject here bc other routes may call authMiddleware without ensurePrimary so no reason to reject those. wdyt

Reply:
assuming we good here

Reply:
ya